packages/core/src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -46,6 +46,10 @@ export interface ProductCardProps {
    * Specifies the button's label.
    */
   buttonLabel?: string
+  /**
+   * Specifies item list name.
+   */
+  list_name?: string
```

Comment:
```suggestion
  listName?: string
```

todo: can you please rename it and update all the occurrences? Just to maintain the consistency in the codebase. Thanks

Reply:
Got fairly messy because the rebase... But it's done!

Reply:
omggg  🙈 
![Uploading image.png…]()


@lucvysk Sorryyy! Do you mind opening a new PR from the main branch with these changes? I’d really appreciate it! 🤩